### PR TITLE
remove knative references from components

### PR DESCRIPTION
**Pull request by k15r**
**Description**

Changes proposed in this pull request:

- remove references to knative-serving from components

**Related issue(s)**

### CODE REVIEW

File: components/console-backend-service/docs/configuration.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Use the following environment variables to configure the application:
 | APP_VERBOSE | No | No | Show detailed logs in the application. |
 | APP_KUBECONFIG_PATH | No | None | The path to the `kubeconfig` file, needed for running an application outside of the cluster. |
 | APP_INFORMER_RESYNC_PERIOD | No | `10m` | The period of time after which the system resynchronizes the informers. |
-| APP_SYSTEM_NAMESPACES | No | `istio-system,knative-eventing,knative-serving,kube-public,kube-system,kyma-installer,kyma-integration,kyma-system,natss,compass-system` | The list of Namespaces that should be considered as system Namespaces. |
+| APP_SYSTEM_NAMESPACES | No | `istio-system,kube-public,kube-system,kyma-installer,kyma-integration,kyma-system,natss,compass-system` | The list of Namespaces that should be considered as system Namespaces. |
```
Comment (nachtmaar) [inline components/console-backend-service/docs/configuration.md:18]: Why did you remove `knative-eventing` ?
```suggestion
| APP_SYSTEM_NAMESPACES | No | `istio-system,knative-eventing,kube-public,kube-system,kyma-installer,kyma-integration,kyma-system,natss,compass-system` | The list of Namespaces that should be considered as system Namespaces. |
```

**Review by alexandra-simeonova — DISMISSED** [review]

**Review by nachtmaar — APPROVED** [review]

**Review by alexandra-simeonova — APPROVED** [review]

**Review by strekm — APPROVED** [review]
@k15r how do you plan to remove this component from existing clusters? will there be follow-up on that? other than that /meow

**Review by aerfio — APPROVED** [review]

**Comment by k15r** [comment]
> @k15r how do you plan to remove this component from existing clusters?
Could you elaborate on which component you mean? If you mean knative-serving itself, it is already gone. This PR is merely a cleanup to remove all knative-serving dependencies from other components binaries.

**Review by kazydek — APPROVED** [review]

**Review by polskikiel — APPROVED** [review]

**Review by a-thaler — APPROVED** [review]

**Review by Maladie — APPROVED** [review]

**Review by kwiatekus — APPROVED** [review]
